### ISSUE TITLE
Problems deploying this application to gh-pages

### ISSUE THREAD

**Issue by adriatic** [aurelia-ui-toolkits/demo-kendo#4]
I finally discovered why was I not able to deploy demo-kendo: it was my interpretation of the instructions in the **[README](https://github.com/aurelia-ui-toolkits/demo-kendo/blob/master/README.md)** file. The actual command that failed is **[deploy.js](https://github.com/aurelia-ui-toolkits/demo-kendo/blob/master/build/tasks/deploy.js)** - which consistently failed to copy **[anything](https://github.com/aurelia-ui-toolkits/demo-kendo/blob/master/build/tasks/deploy.js#L14-L20)** from the jspm_packages folder. 

Once I looked in the content of that folder, I realized that this is indeed the correct behavior - as nothing in that folder matches the definitions below

```
    './jspm_packages/*.js', // include system.js
    './jspm_packages/github/twbs/bootstrap@*/fonts/*',
    './jspm_packages/npm/font-awesome@*/**/*',
    './jspm_packages/npm/font-awesome*.js',
    './jspm_packages/github/PrismJS/prism*/themes/*.css',
    './jspm_packages/github/google/code-prettify*/loader/**/*',
    './jspm_packages/npm/babel-runtime*/**/*',
```

In other words, the jspm_packages was not correct in my situation. So, before going any deeper in this analysis, @JeroenVinke can you please confirm that the instructions in the README.md are still accurate?

**In particular:**
1. What are the prerequisites for the **[How to run](https://github.com/aurelia-ui-toolkits/demo-kendo#how-to-run)** section?. (do I first clone this repo and then start with this section?)
2. Item 2.vi. requires Telerik credentials so, if I do need to run this step, I know what was my "jinks" as I do not have these credentials.
3. The item 2. in **[How to deploy from samples](https://github.com/aurelia-ui-toolkits/demo-kendo#how-to-deploy-from-samples)** is obviously inaccurate as there is no `doc` folder in the `aurelia-kendoui-plugin/sample` folder.

**Comment by JeroenVinke** [comment]
I will update the readme. Please explain what **is** in your jspm_packages folder.

**Comment by adriatic** [comment]
Nothing at the moment as I deleted the local `demo-kendo` folder. At the moment I have just the clone of the original and am waiting for your updated README.md, to finally do this task without a hitch.

**Comment by JeroenVinke** [comment]
updated readme: https://github.com/aurelia-ui-toolkits/demo-kendo/blob/master/README.md

I would suggest to wait a few days before attempting a deploy again, so if it goes haywire again I can push from my PC to fix it

**Comment by adriatic** [comment]
That is a good advice. I will however run the "test" version of the deployment, where the data gets written to the `dist` folder instead of being uploaded. If that works OK, there is no reason for not updating the gh_pages - other than what you just stated above; so I will not deploy, just test

**Comment by adriatic** [comment]
### Update

Using the updated version of the README.md file, I redid the whole process until I reached the very last step **`gulp deploy`** (and push upstream). Everything seems just fine, the demo-app acts as it should - and the only reason I did not finish this is the following **fork error indication**.

```ok   Installed google/code-prettify as github:google/code-prettify@master (master)
ok   Installed core-js as npm:core-js@^1.1.4 (1.2.7)
ok   Installed prism as github:PrismJS/prism@1.3.0 (1.3.0)
     Installed Forks

                                                    npm:isarray 0.0.1 1.0.0

     To inspect individual package constraints, use jspm inspect registry:name.

     Looking up loader files...
       system.js.map
       system.js
       system-csp-production.js
       system-csp-production.js.map
       system.src.js
       system-polyfills.src.js
       system-polyfills.js
       system-csp-production.src.js
       system-polyfills.js.map

     Using loader versions:
       systemjs@0.19.6
ok   Loader files downloaded successfully

ok   Install complete.
[13:58:24] Using gulpfile C:\work\aurelia-ui-toolkits\kendoui\demo-kendo\gulpfile.js
[13:58:24] Starting 'serve'...
[BS] Local URL: http://localhost:9000
[BS] Serving files from: .
[13:58:25] Finished 'serve' after 1.28 s
[13:58:25] Starting 'watch'...
[13:58:25] Finished 'watch' after 6.61 μs
```

That is a very often fork that I was never able to resolve, so I stopped here.

**Comment by Thanood** [comment]
It's `npm:prettify@0.1.7`
- which depends on `js-prettify@1.4.0`
- which depends on `nopt@2.1.2`
- which depends on `nodelibs-stream@0.1.0`
- which depends on `stream-browserify@1.0.0`
- which depends on `readable-stream@1.1.14`
- (which, btw. depends on `stream-browserify@1.0.0` - a circular reference)
- which depends on `isarray@0.0.1`

It seems all to "come down" to this line:
https://github.com/psyrendust/js-prettify/blob/master/package.json#L49
Because there's a newer (major) version of `nodelibs-stream` available which down the dependency line wouldn't import `isarray` at all.

This is at least as far as I could see. 😃 

A lot more packages rely on `nodelibs-stream` so this seems unavoidable (for us).

I suspect this happens alot with lower-level libraries, because some other libraries depend on them which have "restrictive" version ranges.

**Comment by Thanood** [comment]
I think for this particular fork the best solution is to run:
`jspm resolve --only npm:isarray@1.0.0`

**Comment by adriatic** [comment]
@Thanood thank you (you are a better man, as you must have spent some good time to chase this down 😄 ).

`jspm resolve --only npm:isarray@1.0.0` did the job:

```
λ jspm resolve --only npm:isarray@1.0.0
     In npm:readable-stream@1.1.14 0.0.1 -> 1.0.0
ok   Resolution to only use npm:isarray@1.0.0 completed successfully.
```

**Comment by adriatic** [comment]
### http://aurelia-ui-toolkits.github.io/demo-kendo sucessfuly deployed
